editor/src/components/inspector/common/css-utils.ts

```diff
@@ -4836,10 +4836,12 @@ function parseValueFactory<T, K extends keyof T>(parserMap: {
   }
 }
 
-const parseMetadataValue = parseValueFactory(elementPropertiesParsers)
-const parseCSSValue = parseValueFactory(cssParsers)
-const parseOldLayoutValue = parseValueFactory(layoutParsers)
-const parseNewLayoutValue = parseValueFactory(layoutParsersNew)
+const parseMetadataValue = Utils.memoize(parseValueFactory(elementPropertiesParsers), {
+  maxSize: 1000,
```

Comment:
how close does the before-i-go project comes to hitting this maxSize per frame?

Reply:
I'll have a look and see, I set it high because the values involved should be quite small.